src/AuthenticationService.php

```diff
@@ -220,21 +220,18 @@ public function authenticate(ServerRequestInterface $request, ResponseInterface
      * Clears the identity from authenticators that store them and the request
      *
      * @param \Psr\Http\Message\ServerRequestInterface $request The request.
+     * @param \Psr\Http\Message\ResponseInterface The response.
      * @return \Psr\Http\Message\ServerRequestInterface
      */
-    public function clearIdentity(ServerRequestInterface $request, $requestAttribute = 'identity')
+    public function clearIdentity(ServerRequestInterface $request, ResponseInterface $response)
     {
         foreach ($this->_authenticators as $authenticator) {
             if ($authenticator instanceof PersistenceInterface) {
                 $authenticator->clearIdentity($request);
```

Comment:
Yes, forgot to restore that after I accidentally did a git reset... Added it back.

But about the return value: What do you want to return? The authenticators could return bool but what should the service return? The only meaningful thing would be an array of authenticator and status of the call Session => true for example to figure out which and if all of them cleared the identity.